### PR TITLE
Bugfix cannot initialize cluster

### PR DESCRIPTION
**Pull request by jacobcrawford**
* Updated bug in helm chart url
* Temporarily removing support for external certificate generation because of bug with. kubeadm generates on boot and uploads to other nodes. A task has been created to reimplement this feature. 
* Boot.sh was not copied to the worker directory

### CODE REVIEW

File: scripts/prepare_master_certs.sh [hunk 1]
```diff
@@ -9,14 +9,14 @@ mkdir -p $OUTPUT_DIR
...
-        --control-plane-endpoint $NODE_CONTROL_PLANE_VIP:$NODE_CONTROL_PLANE_PORT \
-        --apiserver-cert-extra-sans $MASTER_VIP_CLUSTER_IPS
+    #docker run \
+    #    -v $PWD/$OUTPUT_DIR:/src \
+    #    --user "$(id -u):$(id -g)" \
+    #    kubeadocker \
+    #    init phase certs all \
+    #    --cert-dir /src \
+    #    --control-plane-endpoint $NODE_CONTROL_PLANE_VIP:$NODE_CONTROL_PLANE_PORT \
+    #    --apiserver-cert-extra-sans $MASTER_VIP_CLUSTER_IPS
```
Comment (skovbakke) [inline scripts/prepare_master_certs.sh:19]:
```suggestion

```
If we are sure we do not want it let us delete it.

File: scripts/prepare_cluster.sh [hunk 2]
```diff
@@ -48,8 +48,8 @@ for ((i=1; i<=${#HOSTS[@]}; i++)); do
         export MASTER_PROXY_STATE=MASTER
         export MASTER_CREATE_CLUSTER=true
         # Copy the certs to folder
-        mkdir $OUTPUT_DIR_MASTER/etc/kubernetes/pki -p
-        cp -r build/cluster/certs/* $OUTPUT_DIR_MASTER/etc/kubernetes/pki
+        #mkdir $OUTPUT_DIR_MASTER/etc/kubernetes/pki -p
+        #cp -r build/cluster/certs/* $OUTPUT_DIR_MASTER/etc/kubernetes/pki
```
Comment (skovbakke) [inline scripts/prepare_cluster.sh:52]:
```suggestion

```
This could also be deleted

**Review by skovbakke — CHANGES_REQUESTED** [review]
We should not use comments for code that should not execute. If code should not execute just delete it. We can always go back in the git history to find it.

File: scripts/prepare_master_certs.sh (hunk 1 above)
Comment (jacobcrawford) [reply scripts/prepare_master_certs.sh:19]: Whole file is deleted.

File: scripts/prepare_cluster.sh (hunk 2 above)
Comment (jacobcrawford) [reply scripts/prepare_cluster.sh:52]: Done.

**Review by skovbakke — COMMENTED** [review]
Looks good

**Review by skovbakke — APPROVED** [review]

**Pull request merged**